pkgs/development/python-modules/gurobipy/linux.nix

```diff
@@ -1,21 +1,18 @@
-{ fetchurl, python }:
-assert python.pkgs.isPy27;
+{ lib, fetchurl, python }:
```

Comment:
```suggestion
{ lib, fetchurl, buildPythonPackage }:
```

Reply:
Hmm, `python.libPrefix` is used in an argument to `patchelf`.